registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -501,6 +503,7 @@
     "sha256:e053982bbf4662e2b89235fcae2ee505f6063eb504fad935f34a3b31c4e1af56": ["v1.8.1"]
     "sha256:a83343cafc08756b71a0ae85288914cbbc30986db4bb5647a694a7cd08ddb51f": ["v1.8.10"]
     "sha256:c4e595853469359e669edd99a006b11a0171a2eee3d471805d383ae9291ca8d4": ["v1.8.11"]
+    "sha256:5ad3e4cc78a66c70e56d0fe835db0ee645e19d71f135613068bae9749e23437d": ["v1.8.12"]
```

Comment:
`5ad3e4cc78a6` matches the SHA for kubeadm-bootstrap-controller v1.8.12 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:5ad3e4cc78a66c70e56d0fe835db0ee645e19d71f135613068bae9749e23437d/details). 👍🏻